### PR TITLE
sarif-sdk: Update text for verbose, recurse, quiet, environment, rich-return-code

### PR DESCRIPTION
**Pull request by shaopeng-gh**
The other part is BinSkim:
https://github.com/microsoft/binskim/pull/880

### CODE REVIEW

File: src/Sarif/SdkResources.resx [hunk 1]
```diff
@@ -226,10 +226,10 @@
     <value>Unexpected fatal runtime condition(s) observed: </value>
   </data>
   <data name="MSG_OneOrMoreInvalidTargets" xml:space="preserve">
-    <value>One or more targets was skipped entirely as it was determined to be an invalid target for analysis. Pass --verbose on the command-line for more information.</value>
+    <value>One or more targets was skipped entirely as it was determined to be an invalid target for analysis. Include all supported values for both --kind and --level on the command-line for more information.</value>
```
Comment (michaelcfanning) [inline src/Sarif/SdkResources.resx:229]: >Include all supported values for both --kind and --level [](http://example.com/codeflow?start=111&length=56)

I believe that `--kind Fail;NotApplicable` is the specific value required. You don't need `--level` here as this severity level isn't relevant except for kind == Fail. Can you test this?

Here's a new sentence for you if I'm correct in what I assume:

Include `NotApplicable` on an explicit `--kind` command-line argument (e.g., `--kind Fail;NotApplicable`) for more information.

Comment (shaopeng-gh) [reply src/Sarif/SdkResources.resx]: tested and updated. 
MSG_OneOrMoreInvalidTargets and 
MSG_OneOrMoreNotApplicable is different,
I have tested the specific value required.

Comment (shaopeng-gh) [reply src/Sarif/SdkResources.resx]: oh btw I did not use the markup backtick, we don't use markup in the console output currently and i think that is correct.

**Review by suvamM — APPROVED** [review]
:shipit:

**Comment by shaopeng-gh** [comment]
@michaelcfanning @suvamM 
Please review with this together to get the full change I am making:
https://github.com/microsoft/sarif-sdk/pull/2661

File: docs/multitool-usage.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Sarif.Multitool export-validation-config validation.xml
 Sarif.Multitool export-validation-rules ValidationRules.md
 
 : Merge multiple SARIF files into one
-Sarif.Multitool merge C:\Input\*.sarif --recurse --output-directory=C:\Output\ --output-file=MergeResult.sarif
+Sarif.Multitool merge C:\Input\*.sarif --recurse true --output-directory=C:\Output\ --output-file=MergeResult.sarif
```
Comment (michaelcfanning) [inline docs/multitool-usage.md:59]: >true [](http://example.com/codeflow?start=49&length=4)

This shouldn't be literally required, you *can* provide an explicit true but you can also exclude it. Do you not observe that this is correct in your testing?

File: src/Test.UnitTests.Sarif/HashUtilitiesTests.cs [hunk 3]
```diff
@@ -72,7 +72,7 @@ public void HashUtilities_ShouldRespectQuietFlag()
                 //  Do nothing, this is the desired behavior.
             }
 
-            //  Mimic the --quiet flag.  We should not use the console this time.
+            //  Mimic the '--quiet true'.  We should not use the console this time.
```
Comment (michaelcfanning) [inline src/Test.UnitTests.Sarif/HashUtilitiesTests.cs:75]: >quiet [](http://example.com/codeflow?start=29&length=5)

Again all of these values s/b legal:

--quiet (s/be interpreted as --quiet true)
-- quiet true
--quiet false

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Pull request merged**